src/Sulu/Component/Content/Document/Structure/PropertyValue.php

```diff
@@ -87,7 +87,7 @@ public function offsetSet($offset, $value)
             return;
         }
 
-        $this->value[$offset] = array_merge($this->value[$offset], $value);
+        $this->value[$offset] = $value;
```

Comment:
Looks reasonable to me. I am just wondering why it has been different in the first place.